src/client/components/Layout/Layout.js

```diff
@@ -53,3 +53,5 @@ Layout.propTypes = {
 };
 
 export default withRouter(Layout);
+
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the end of file. Max of 1 allowed.](https://app.codacy.com/gh/ShipyardSuite/dashboard/pullRequest?prid=5673418)